lib/splitclient-rb/engine/parser/evaluator.rb

```diff
@@ -2,41 +2,60 @@ module SplitIoClient
   module Engine
     module Parser
       class Evaluator
-        def initialize(segments_repository, splits_repository, config, multiple = false)
+        def initialize(segments_repository, splits_repository, config)
           @splits_repository = splits_repository
           @segments_repository = segments_repository
-          @multiple = multiple
           @config = config
           @cache = {}
         end
 
-        def call(keys, split, attributes = nil)
+        def evaluate_feature_flag(keys, feature_flag, attributes = nil)
```

Comment:
i think that you should get the `Feature Flag definition` here and not in the client to be consistency between methods